Feature/cards and boxes using bslib

aj-thl opened the pull request:
Implemented cards and boxes by using bslib.

tinneuro commented:
Great thanks a lot I will do the review today or tomorrow :)

tinneuro commented on NEWS.md:
```diff
@@ -26,6 +26,8 @@
 * Improved wording of objects in signals and reports tab
 * Made table appearance for interactive and static tables consistent using new dependency `flextable`
 * Recodes NAs in signal tables to 'unknown' for all stratifications but not for the unstratified level
+* Changed the title of the tool to "Signal Detection Tool".
```
Can you add these news to the "new section 0.2.1"? I added now in develop because 0.2.0 was already published

tinneuro commented:
Furthermore shinyWidgets needs to be added to the DESCRIPTION Imports

tinneuro commented:
I really like the new style!! It is great. Thanks a lot for your work!

aj-thl commented:
Thanks @tinneuro! It's nice to hear that you like the changes 😄 I updated the NEWS and the DESCRIPTION as you requested. I also noticed that pr was merging into main instead of the develop branch so I also corrected this by changing the base branch from main to develop. You may need to do the review again, my apologies for this.

tinneuro commented:
Great no worries. I was anyways not finished yet

tinneuro commented:
I found one bug: when using place_of_infection as strata then we get an error and nothing is shown. Maybe you can shortly investigate this?

tinneuro commented on NEWS.md:
```diff
@@ -27,9 +32,6 @@
 * Allow selection of all character and factor variables in the dataset, except for the variables ending with '_id', to use for filters and stratifications.
 * Adding a function `pad_signals()` performing the padding of signals timeseries (extension of threshold and expecation to the past for visualisation) which was previously done in the signals tab server.
 * adding method and number_of_weeks to the resulting tibble of `get_signals()` to reduce number of parameters needed to be passed forward.
-* Improved wording of objects in signals and reports tab
```
Probably just a volatility error: can you move the deleted points back to 0.2.0 and just add your new points to 0.2.1? Thanks

aj-thl replied:
Sure!

aj-thl commented:
What kind of an error you are getting when using place_of_infection as strata? I tried this (other input parameters in default values) and I could not reproduce the error. I put a screenshot of the results below:
![image](https://github.com/United4Surveillance/signal-detection-tool/assets/141014269/f034583b-5afb-4676-858f-6995448e0213)

aj-thl commented:
The NEWS has now been corrected.

tinneuro commented:
Somehow the error disappeared which is great :D

tinneuro commented:
I am sorry, I just needed to quickly update something on develop. so now you need to change the NEWS one last time

tinneuro commented:
I finished reviewing everything so after this resolve I will approve the PR :)

tinneuro reviewed (approved):
Thank you! Great improvments.
